scss/_navbar.scss

```diff
@@ -1,29 +1,37 @@
-// Contents
-//
-// Navbar
-// Navbar brand
-// Navbar nav
-// Navbar text
-// Responsive navbar
-// Navbar position
-// Navbar themes
-
-
 // Navbar
 //
 // Provide a static navbar from which we expand to create full-width, fixed, and
 // other navbar variations.
 
 .navbar {
+  // scss-docs-start navbar-css-vars
+  --#{$variable-prefix}navbar-padding-x: #{if($navbar-padding-x == null, 0, $navbar-padding-x)};
+  --#{$variable-prefix}navbar-padding-y: #{$navbar-padding-y};
+  --#{$variable-prefix}navbar-color: #{$navbar-light-color};
+  --#{$variable-prefix}navbar-hover-color: #{$navbar-light-hover-color};
+  --#{$variable-prefix}navbar-disabled-color: #{$navbar-light-disabled-color};
+  --#{$variable-prefix}navbar-active-color: #{$navbar-light-active-color};
+  --#{$variable-prefix}navbar-brand-padding-y: #{$navbar-brand-padding-y};
+  --#{$variable-prefix}navbar-brand-margin-end: #{$navbar-brand-margin-end};
+  --#{$variable-prefix}navbar-brand-color: var(--#{$variable-prefix}navbar-active-color);
+  --#{$variable-prefix}navbar-brand-hover-color: var(--#{$variable-prefix}navbar-active-color);
```

Comment:
By using CSS variables for the two brand color values, you're making the SCSS variables for both light and dark brand color and brand hover colors not have any purpose. If you're trying to avoid breaking changes, then I would set them to their existing light SCSS variables values here, and then make sure to override them in `.navbar-dark` with their dark variants. Otherwise we should just get rid of those SCSS variables completely since they won't be for anything anymore, but that's for sure even more of a breaking change.

Reply:
Yeah, adding this to the todos—I left a commit in another local branch and forgot to note it here. Thanks for flagging!